src/mono/sample/wasm/blazor-frame/wwwroot/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <title>blazor</title>
-    <!-- <base href="/blazor-template/" /> -->
```

Comment:
Just curious, what's the reason for this change?

Reply:
The `Index.razor` page wasn't loading properly, so I changed it according to https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/?view=aspnetcore-8.0&tabs=visual-studio#app-base-path

Interestingly I had to also change the `blazor.webassembly.js` path to make it work.